src/neuroglancer/sliceview/png/LICENSE

```diff
@@ -0,0 +1,206 @@
+The following license covers all code in this png directory
```

Comment:
Since the third-party code is no longer copied here, we can exclude this additional license file.